process-document/src/main/java/com/ritense/processdocument/security/config/ProcessDocumentHttpSecurityConfigurer.java

```diff
@@ -48,21 +48,31 @@ public void configure(HttpSecurity http) {
                 .authenticated()
                 .requestMatchers(antMatcher(POST, "/api/v1/process-document/operation/new-document-and-start-process"))
                 .authenticated()
-                .requestMatchers(antMatcher(POST, "/api/v1/process-document/operation/modify-document-and-complete-task"))
+                .requestMatchers(antMatcher(
+                    POST,
+                    "/api/v1/process-document/operation/modify-document-and-complete-task"
+                ))
                 .authenticated()
-                .requestMatchers(antMatcher(POST, "/api/v1/process-document/operation/modify-document-and-start-process"))
+                .requestMatchers(antMatcher(
+                    POST,
+                    "/api/v1/process-document/operation/modify-document-and-start-process"
+                ))
                 .authenticated()
                 .requestMatchers(antMatcher(POST, "/api/v3/task"))
                 .authenticated()
                 .requestMatchers(antMatcher(POST, "/api/v1/document-definition/{caseDefinitionName}/task/search"))
                 .authenticated()
                 //admin endpoints
-                .requestMatchers(antMatcher(GET, FEATURE_PROCESS_URL+"/{type}"))
+                .requestMatchers(antMatcher(GET, FEATURE_PROCESS_URL + "/{type}"))
                 .hasAuthority(ADMIN)
                 .requestMatchers(antMatcher(PUT, FEATURE_PROCESS_URL))
                 .hasAuthority(ADMIN)
-                .requestMatchers(antMatcher(DELETE, FEATURE_PROCESS_URL+"/{type}"))
+                .requestMatchers(antMatcher(DELETE, FEATURE_PROCESS_URL + "/{type}"))
                 .hasAuthority(ADMIN)
+                .requestMatchers(antMatcher(
+                    PUT,
+                    "/api/management/v1/case-definition/{caseDefinitionKey}/version/{caseDefinitionVersionTag}/process/{processDefinitionId}"
```

Comment:
Maybe add a postfix like `/association` so that it doesn't look like it will update the entire process